fern/docs.yml

```diff
@@ -6,64 +6,193 @@ instances:
+            path: ./overview/dummy.mdx
+      - section: Writing Content
+        collapsed: true
+        contents:
+          - page: Page Titles and Metadata
+            path: ./overview/dummy.mdx
+          - page: Headers and Text
+            path: ./overview/dummy.mdx
+          - page: Images, Videos and Embeds
+            path: ./overview/dummy.mdx
+          - page: Lists and Tables
+            path: ./overview/dummy.mdx
+          - page: Code Blocks
+            path: ./overview/dummy.mdx
+      - section: API References
+        collapsed: true
+        contents:
+          - page: API Definition Setup
+            path: ./overview/dummy.mdx
+          - page: Errors, Audiences, and Snippets
+            path: ./overview/dummy.mdx
+          - page: API Playground
+            path: ./overview/dummy.mdx
+      - section: Custom Components
+        collapsed: true
+        contents:
+          - page: Accordion
+            path: ./overview/dummy.mdx
+          - page: Availability
+            path: ./overview/dummy.mdx
+          - page: Callout
+            path: ./overview/dummy.mdx
+          - page: Cards
+            path: ./overview/dummy.mdx
+          - page: Code Blocks
+            path: ./overview/dummy.mdx
+          - page: Endpoint Snippets
+            path: ./overview/dummy.mdx
+          - page: Icon
+            path: ./overview/dummy.mdx
+          - page: Steps
+            path: ./overview/dummy.mdx
+          - page: Tabs
+            path: ./overview/dummy.mdx
+  - tab: config
+    layout:
+      - section: Fern Folder
+        contents:
+          - page: Overview
+            path: ./overview/dummy.mdx
+          - page: generators.yml
+            path: ./overview/dummy.mdx
+          - page: docs.yml
+            path: ./overview/dummy.mdx
+      - section: API Definitions
         contents:
-          # TBD
-          # - page: Overview
-          #   path: ./docs/pages/overview/define-your-api/api-overview.mdx
-          - section: OpenAPI
-            slug: openapi
+          - section: OpenAPI Spec
             contents:
               - page: Overview
-                path: ./docs/pages/overview/define-your-api/openapi/openapi-overview.mdx
-              - page: OpenAPI Extensions
-                path: ./docs/pages/overview/define-your-api/openapi/extensions.mdx
-                slug: extensions
-              - page: OpenAPI Examples
-                path: ./docs/pages/overview/define-your-api/openapi/examples.mdx
-                slug: examples
+                path: ./overview/dummy.mdx
+              - section: Extensions
+                contents:
+                  - page: Overriding Names
+                    path: ./overview/dummy.mdx
+                  - page: Auth and Global Headers
+                    path: ./overview/dummy.mdx
+                  - page: Audiences
+                    path: ./overview/dummy.mdx
+                  - page: Ignoring Endpoints
```

Comment:
In general, I recommend avoiding gerunds/participles as the first word in headers/menus (`-ing` words like `Writing` `Ignoring` `Overlaying` `Merging` etc) if the more concise imperative form will do. 

As in: `Write content`, `Ignore endpoints`, `Merge multiple APIs`